app/ui.js

```diff
@@ -1714,6 +1701,21 @@ const UI = {
  */
 };
 
-export default UI;
+// Set up translations
+var LINGUAS = ["de", "el", "nl", "sv"];
+l10n.setup(LINGUAS);
+if (l10n.language !== "en" && l10n.dictionary === undefined) {
+    WebUtil.fetchJSON('app/locale/' + l10n.language + '.js', function (translations) {
+        l10n.dictionary = translations;
+
+        // wait for translations to load before loading the UI
+        UI.load();
+    }, function (err) {
+        throw err; 
+    });
+} else {
+    UI.load();
+}
+
 
-UI.load();
```

Comment:
I think you mentioned this in an earlier commit, so you may want to tweak the messages. :)